channeldb/channel.go

```diff
@@ -818,6 +822,10 @@ type OpenChannel struct {
 	// default ShortChannelID. This is only set for zero-conf channels.
 	confirmedScid lnwire.ShortChannelID
 
+	// sentShutdown denotes whether or not we've sent the Shutdown message
+	// for this channel.
+	sentShutdown bool
```

Comment:
Can alternatively store a _nested_ TLV here that can be extended to store the: delivery script sent, fee range, last offer, etc.